data/hai/hai reveal 4 middle.txt

```diff
@@ -1413,13 +1413,122 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	"Security through secrecy..." He coughs. "You throw that in there as if it were always so Gemnali but, unlike you, I was there when it was added to the mandate of this chamber."`
+			`	He peers around the room and you realize he is blind, or nearly so. "Freedom may be achieved through peace, but peace must not be maintained at the cost of freedom. Order may not be pursued at the cost of peace, nor security at the cost of order. Always this mandate must be considered in both directions. To censure this woman, to resist action on this matter. You would have us pursue secrecy at the cost of security."`
+			`	He coughs again it echoes in the silence of nearly a hundred Hai holding their breath. When he continues his voice has grown stronger and more cutting. "My eyes may not see as well as they used to, but there are many in this chamber who are yet too blind to see. The edifice of our stability is not threatened by this woman who would have us restore security to our lost brethren. It is you who would use her virtue to topple it yourself."`
+			`	The silence stretches with all eyes on Gemnali. She clenches her fists, shaking with barely suppressed emotion, before collapsing back into her seat as if physically struck.`
+			`	Whatever spell had held the room is immediately broken and a susurrus of murmuring rises to fill its space. Within the noise a certain phrase starts to be repeated multiple times, drawing an expression of shock from Sayari.`
+			choice
+				`	"What are they saying?"`
+				`	"What just happened?"`
+			`	She glances at you briefly. "They are saying; Veeheylur and Misanaya fight over Eyrisen." Her tone carries with it a sense of awe and despite the growing volume in the room she leans in close to you and continues in scarcely more than a whisper. "The mythos has been invoked."`
+			`	As Aimet painfully resumes his seat, one Elder who has thus far been maintaining a fairly neutral demeanor in this seems to regain a certain sense of himself and stands up, quickly heralded by the moderator who struggles to compose himself suitably. "I think the council has heard enough. The petitioner is entitled to a closing statement unless there is objection?"`
+			`	None of those assembled who might object seem to have recovered their composure enough to stand up quickly and the moderator, perhaps sensing a politic moment to intervene, quickly announces the closing of the interrogation and calls the council to hear Teenep again.`
+			`	Teeneep quickly plasters her mask of calm back on as best she is able and says, "Thank you all for your ear. You are those we trust to decide, but know this. If you will not listen to this here, then apparently I must now be the one to present a pathway to avoid war. The Unfettered are committed no matter what path I choose now. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`
```

Comment:
```suggestion
			`	Teeneep quickly plasters back on her mask of calm as best she is able to and says, "Thank you all for your ears. You are those we trust to decide, but know this: If you will not listen to this here, then apparently I must now be the one to present a pathway to avoid war. The Unfettered are committed, no matter what path I choose now. If you do not take action, I will make the history of inaction I have discovered public knowledge. Every Hai from Greenwater to Snowfeather will know by the end of the day."`
```
From Ireland to Cathay.